lib/league_stats.rb

```diff
@@ -0,0 +1,139 @@
+require 'csv'
+require_relative 'stats'
+require_relative 'game'
+require_relative 'team'
+require_relative 'game_teams'
+
+class LeagueStats < Stats
+  attr_reader :games, :teams, :game_teams
+
+  def initialize(data)
+    @games = build_games(data)
+    @teams = build_teams(data)
```

Comment:
Great readability and coder empathy when writing methods. Easily readable and understandable.

Reply:
Thank you, Nick helped me write a lot of this!